api/bazel/repository_locations.bzl

```diff
@@ -7,8 +7,8 @@ GOGOPROTO_SHA256 = "99e423905ba8921e86817607a5294ffeedb66fdd4a85efce5eb2848f715f
 OPENCENSUS_PROTO_GIT_SHA = "d5d80953a8c2ff4633087af6933cd152678434bb"  # Mar 18, 2019
 OPENCENSUS_PROTO_SHA256 = "a4e87a1da21d1b3a16674332c3ee6e2689d52f3532e2ff8cb4a626c8bcdabcfc"
 
-PGV_GIT_SHA = "26db5cb7c01a67c6a2e21a832106406185530b2f"
-PGV_SHA256 = "6510cbcf69d99059c652ae2376f6240bc761d0b019cd962225f4f609be361e26"
+PGV_GIT_SHA = "dd90514d96e35023ed20201f349542b0bc64461d"
```

Comment:
@SecurityInsanity Would have been nice to refer to the commit date of July 19 2019 like we do with other non-release dependencies. As a TODO we should switch to PGV release 1.1 when it is released and assuming it contains this commit. Sorry I didn't pick this up earlier.

Reply:
Hey @moderation , yeah we should switch to more stabilized versions, and comment for sure. However, keeping status quo here I don't think is particularly bad. 

Something to remember, but not all that important for now.